mne_connectivity/base.py

```diff
@@ -662,8 +662,9 @@ def save(self, fname):
 
         # save as a netCDF file
         # note this requires the netcdf4 python library
-        self.xarray.to_netcdf(fname, mode='w', format='NETCDF4',
-                              engine='netcdf4')
+        self.xarray.to_netcdf(fname, mode='w',
+                              format='NETCDF4',
+                              engine='h5netcdf')
```

Comment:
Is specifying the engine required? Does one work and the other not, or is one just slow, or...? A comment here will help maintainability down the road as engines may change and downstream libraries might get better or worse

Reply:
Specifyinng the engine is required as currently `h5netcdf` is the only engine that supports writing complex data. Xarray typically didn't include support for complex data, so their NetCDF engine doesn't work without some hacks :/

I'll add an inline comment